FileIO/AsciiRasterInterface.h

```diff
@@ -49,6 +52,10 @@ class AsciiRasterInterface {
                                  double &min, double &max);
 };
 
+/// Reads a vector of rasters given by file names. On error nothing is returned,
+/// otherwise the returned vector contains pointers to the read rasters.
+boost::optional<std::vector<GeoLib::Raster const*>> readRasters(
```

Comment:
it looks bit redundant for me to use `boost::optional` for `std::vector`, because one can do the same thing using `std::vector::empty()`.